src/commands/deployAzurePipeline/configureCicdPipeline/models/resourceNode.ts

```diff
@@ -0,0 +1,6 @@
+import { GenericResource } from 'azure-arm-resource/lib/resource/models';
+
+export interface resourceNode extends GenericResource{

```

Comment:
💡 **Question**: Why we need `extend` here? because we are `annotating` property -> `resource` as `GenericResource`.

Reply:
yeah, we need nto extend